docs/bucket/notifications/README.md

```diff
@@ -840,13 +840,13 @@ To update the configuration, use `mc admin config get` command to get the curren
 
 ```sh
 $ mc admin config get myminio/ notify_kafka
-notify_kafka:1 tls_skip_verify="off"  queue_dir="" queue_limit="0" sasl_enable="off" sasl_password="" sasl_username="" tls_client_auth="0" tls_enable="off" brokers="" topic=""
```

Comment:
rename sasl_enable to sasl and tls_enable to tls

Reply:
as you wish) i'm ok with both

Reply:
done in [c3e2365](https://github.com/minio/minio/pull/8593/commits/c3e23659564a6d7cc21b86f6fed033da8b2208cd)